bench/real_client.rb

```diff
@@ -0,0 +1,35 @@
+require "nats/io/client"
+require "thread"
+require "benchmark/ips"
+require "protobuf"
+require 'protobuf/message'
+require 'protobuf/rpc/service'
```

Comment:
in order to make this run locally did the following diff to this file

```diff
-
-module Warehouse
-  class Shipment < ::Protobuf::Message
-    optional :string, :guid, 1
-    optional :string, :address, 2
-    optional :double, :price, 3
-    optional :string, :package_guid, 4
-  end
-end
+require "./examples/warehouse/app"
```

And I forgot to set the PB_CLIENT_TYPE and PB_SERVER_TYPE environment variables, so maybe we should set those at the top of this script just to make it easy for people to run the benchmark.

Locally I got ~1k request/reply's per second. Is that similar to your results so far?